creds/accessbox/accessbox.go

```diff
@@ -19,6 +19,14 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	hkdfSaltLength = 8
```

Comment:
Let's use 16, even though 8 is kinda sufficient, most people take more like https://en.wikipedia.org/wiki/PBKDF2

Reply:
Ok